src/app-layout/visual-refresh/drawers.tsx

```diff
@@ -0,0 +1,328 @@
+      className={clsx(
+        styles['drawers-desktop-triggers-container'],
+        testutilStyles['drawers-desktop-triggers-container'],
+        {
+          [styles['has-multiple-triggers']]: hasMultipleTriggers,
+          [styles['has-open-drawer']]: hasOpenDrawer,
+        }
+      )}
+    >
+      <div
+        className={clsx(styles['drawers-trigger-content'], {
+          [styles['has-multiple-triggers']]: hasMultipleTriggers,
+          [styles['has-open-drawer']]: hasOpenDrawer,
+        })}
+      >
+        {!toolsHide && tools && (
+          <TriggerButton
+            ariaLabel={ariaLabels?.toolsToggle}
+            className={clsx(styles['drawers-trigger'], testutilStyles['tools-toggle'])}
+            iconName="status-info"
+            onClick={() => {
+              activeDrawerId && handleDrawersClick(null, true);
+              handleToolsClick(!isToolsOpen);
+            }}
+            ref={toolsRefs.toggle}
+            selected={isToolsOpen}
+          />
+        )}
+
+        {drawers.items.map((item: DrawersProps.Drawer) => (
+          <TriggerButton
+            ariaLabel={item.ariaLabels?.triggerButton}
+            className={clsx(styles['drawers-trigger'], testutilStyles['drawers-trigger'])}
+            iconName={item.trigger.iconName}
+            iconSvg={item.trigger.iconSvg}
+            key={item.id}
+            onClick={() => {
+              isToolsOpen && handleToolsClick(!isToolsOpen, true);
+              handleDrawersClick(item.id);
+            }}
+            ref={item.id === previousActiveDrawerId.current ? drawersRefs.toggle : undefined}
+            selected={item.id === activeDrawerId}
+          />
+        ))}
+
+        {hasSplitPanel && splitPanelToggle.displayed && (
+          <TriggerButton
+            ariaLabel={splitPanelToggle.ariaLabel}
+            className={clsx(styles['drawers-trigger'], splitPanelStyles['open-button'])}
+            iconName="view-vertical"
+            onClick={() => handleSplitPanelClick()}
+            selected={hasSplitPanel && isSplitPanelOpen}
+            ref={splitPanelRefs.toggle}
+          />
+        )}
+      </div>
+    </aside>
+  );
+}
+
+/**
+ * The MobileTriggers will be mounted inside of the AppBar component and
+ * only rendered when Drawers are defined in mobile viewports. The same logic
+ * will in the AppBar component will suppress the rendering of the legacy
+ * trigger button for the Tools drawer.
+ */
+export function MobileTriggers() {
+  const {
+    activeDrawerId,
+    ariaLabels,
+    drawers,
+    drawersRefs,
+    handleDrawersClick,
+    handleToolsClick,
+    hasDrawerViewportOverlay,
+    isMobile,
+    isToolsOpen,
+    tools,
+    toolsHide,
+    toolsRefs,
+  } = useAppLayoutInternals();
+
+  const previousActiveDrawerId = useRef(activeDrawerId);
+
+  if (activeDrawerId) {
+    previousActiveDrawerId.current = activeDrawerId;
+  }
+
+  if (!isMobile || !drawers) {
+    return null;
+  }
+
+  return (
+    <aside
```

Comment:
I am not sure about rendering a single wrapping `<aside>` for all buttons. The original idea was that each trigger has its own `<aside>` to represent a panel (which is also an `<aside>`).

Reply:
For visibility we discussed this today and there was a general approval from the a11y team for beta. Some feedback suggested that we use a single aside landmark for SplitPanel, Drawers, Tools, and associated trigger buttons. It is unclear whether known issues with a shared landmark have been resolved. This will be addressed in a follow up PR where we share a single implementation for Tools and Drawers.